sources/libraries/bid/c_convert/bid_convert_c.c

```diff
@@ -0,0 +1,50 @@
+#include <bid/bid.h>
+#include <bid/c_convert/bid_convert_c.h>
+#include <bid/c_convert/bid_convert_errors.h>
+#include <bid/c_convert/bid_convert_header.h>
+#include <bid/c_convert/bid_convert_message_type.h>
+#include <bid/c_convert/bid_convert_method.h>
+#include <bid/c_convert/bid_convert_type.h>
+#include <bid/c_convert/bid_convert_typedef.h>
```

Comment:
All these should be a single header file like <bid/c_convert.h> most of them contains a single line a declaration anyway